index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <a href="#"><h1>My Logo</h1></a>
+        <a href="#">
+            <img src="images/Union.png" alt="Union Image">
+        </a>
+    </header>
```

Comment:
- [ ] It appears you have misunderstood the previous reviewer. For semantic reasons, your navbar should contain the `nav` tag. Here, you should still maintain the **`nav`** tag as it was but wrap everything in a **`header`**, like in the snippet below 👇 👇.
```
<header>
  <nav>
    ...
  </nav>
</header>
```

Reply:
Done.